content/02.main-text.md

```diff
@@ -73,14 +73,28 @@ In addition, Manubot relies on extensions from [Pandoc markdown](https://pandoc.
 
 Manubot includes an additional layer of citation processing, currently unique to the system.
 All citations point to a standard identifier, for which Manubot automatically retrieves bibliographic metadata.
-Currently, citations to DOIs (Digital Object Identifiers), PubMed or PubMed Central identifiers, arXiv identifiers, and URLs (web addresses) are supported.
-Metadata is retrieved using DOI [Content Negotiation](https://citation.crosscite.org/docs.html), NCBI's [E-utilities](https://www.ncbi.nlm.nih.gov/books/NBK25501/) and [Citation Exporter](https://www.ncbi.nlm.nih.gov/pmc/tools/ctxp/), the [arXiv API](https://arxiv.org/help/api/index), and [Greycite](http://greycite.knowledgeblog.org/) [@arxiv:1304.7151v1].
-Metadata is exported to [CSL JSON Items](http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html#items), an open standard that's widely supported by reference managers [@doi:10.1007/978-3-319-00026-8_8; @doi:10.1080/02763869.2012.641841].
+Table @tbl:citations presents the supported identifiers and example citations before and after Manubot processing.
+Authors can optionally define citation tags to provide short readable alternatives to the citation identifiers.
+Metadata is exported to [Citation Style Language (CSL) JSON Items](http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html#items), an open standard that is widely supported by reference managers [@doi:10.1007/978-3-319-00026-8_8; @doi:10.1080/02763869.2012.641841].
 In cases where automatic retrieval of metadata fails or produces incorrect references --- which is most common for URL citations --- users can manually provide the correct CSL JSON.
 
-The Manubot formats bibliographies according to a [Citation Style Language](http://citationstyles.org/) (CSL) specification.
+| Identifier | Metadata source | Example citation | Processed citation |
+| --- | --- | --- | --- |
+| Digital Object Identifier (DOI) | DOI [Content Negotiation](https://citation.crosscite.org/docs.html) | `doi:10.1098/rsif.2017.0387` | [@doi:10.1098/rsif.2017.0387] |
+| PubMed Identifier (PMID) | NCBI's [E-utilities](https://www.ncbi.nlm.nih.gov/books/NBK25501/) | `pmid:25851694` | [@pmid:25851694] |
+| PubMed Central Identifier (PMCID) | NCBI's [Citation Exporter](https://www.ncbi.nlm.nih.gov/pmc/tools/ctxp/) | `pmcid:PMC4719068` | [@pmcid:PMC4719068] |
+| arXiv identifier | [arXiv API](https://arxiv.org/help/api/index) | `arxiv:1502.04015v1` | [@arxiv:1502.04015v1] |
+| URL | [Greycite](http://greycite.knowledgeblog.org/) [@arxiv:1304.7151v1] | `url:https://lgatto.github.io/open-and-open/` | [@url:https://lgatto.github.io/open-and-open/] |
+| Tag | Source for tagged identifier | `tag:Avasthi2018_preprints` | [@tag:Avasthi2018_preprints] |
+
+Table: Citation types supported by Manubot.
+Authors map each tag to one of the other supported identifier types.
```

Comment:
Should this sway "Authors _may_ map..." I think the citation by permanent ID is a major benefit and making sure that someone scanning the legend doesn't get confused on the optional tag syntax seems helpful.

Reply:
Updated in e7add1b